.github/workflows/msbuild.yml

```diff
@@ -67,3 +72,109 @@ jobs:
       # This is the best way to fix an issue in master that was only caught by the nightly build.
       all_platforms: ${{ contains(github.event.pull_request.labels.*.name, 'run-deep-tests') || contains(github.event.push.labels.*.name, 'run-deep-tests')}}
       num_shards: 5
+
+  test-coverage-analysis:
+    runs-on: ubuntu-20.04
+    # Don't include in run-deep-tests, because the way it currently runs
+    # messes with integration test coverage tracking. Hopefully running
+    # integration tests as xUnit tests will help avoid this issue.
```

Comment:
`run-deep-tests` still runs them as xUnit tests FYI, the issue is probably related to running from a packaged release of Dafny instead of directly on the source projects.

Reply:
Yes, I think you're right, that the issue is related to running from a packaged release. And I misstated my comment: I was really referring to the change of running them all within the same process. I'll update the comment as part of a merge with `master`.